tools/libs/light/libxl_pci.c

```diff
@@ -754,6 +756,7 @@ bool libxl__is_igd_vga_passthru(libxl__gc *gc,
 
 static int pciback_dev_is_assigned(libxl__gc *gc, libxl_device_pci *pci)
 {
+#if 0
```

Comment:
Don't disable code with preprocessor. Just remove it.

Anyways, git remember everything :)

Reply:
Sorry. Corrected